packages/core-data/src/dynamic-entities.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Internal dependencies
+ */
+import type { GetRecordsHttpQuery, State } from './selectors';
+import type * as ET from './entity-types';
+
+export type WPEntityTypes< C extends ET.Context = 'edit' > = {
+	Comment: ET.Comment< C >;
+	Media: ET.Attachment< C >;
+	Menu: ET.NavMenu< C >;
+	MenuItem: ET.NavMenuItem< C >;
+	MenuLocation: ET.MenuLocation< C >;
+	Plugin: ET.Plugin< C >;
+	PostType: ET.Type< C >;
+	Revision: ET.PostRevision< C >;
+	Sidebar: ET.Sidebar< C >;
+	Site: ET.Settings< C >;
+	Status: ET.PostStatusObject< C >;
+	Taxonomy: ET.Taxonomy< C >;
+	Theme: ET.Theme< C >;
+	UnstableBase: ET.UnstableBase< C >;
+	User: ET.User< C >;
+	Widget: ET.Widget< C >;
+	WidgetType: ET.WidgetType< C >;
```

Comment:
Are these all the entity types? Looks like Global Styles are missing for some reason?

Reply:
Yeah, for some reason, Global Styles seems to be broken. It always returns 404. 

`await wp.data.resolveSelect('core').getGlobalStyles('twentytwentyfive');`

Reply:
Added in 1f5bd6795fe050bd4cbc3f0722d349617854186b

Reply:
Not sure how the plural one is supposed to work 🤔 Return all entries from the `wp_global_styles` post type? I'm not sure this will ever be needed. Maybe the plural just needs to be disabled?

Reply:
But that plural IS actually available on runtime.

Reply:
I have disabled that plural for now.